src/patternfly/components/List/list.scss

```diff
@@ -1,11 +1,26 @@
 .pf-c-list {
+  // list
   --pf-c-list--PaddingLeft: var(--pf-global--spacer--lg);
   --pf-c-list--nested--MarginTop: var(--pf-global--spacer--sm);
   --pf-c-list--nested--MarginLeft: var(--pf-global--spacer--sm);
   --pf-c-list--ul--ListStyle: var(--pf-global--ListStyle);
   --pf-c-list--li--MarginTop: var(--pf-global--spacer--sm);
+
+  // inline
   --pf-c-list--m-inline--li--MarginRight: var(--pf-global--spacer--lg);
 
+  // bordered
+  --pf-c-list--m-bordered--li--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-list--m-bordered--li--BorderBottomColor: var(--pf-global--BorderColor--100);
+  --pf-c-list--m-bordered--li--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+
+  // icon
+  --pf-c-list__item-icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-list__item-icon--Color: var(--pf-global--icon--Color--light);
+  --pf-c-list__item-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+  --pf-c-list__item-icon--m-display-lg--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-list__item-icon--m-display-lg--FontSize: var(--pf-global--icon--FontSize--lg);
```

Comment:
If this is just a larger sized icon for general use, this can probably just be `pf-m-lg`. If it's specifically for marketing use the `pf-m-display-lg` modifier is appropriate.

Also is it the case that if a user opts in to large icons, that should apply to the entire list? Or could you mix and match small and large icons?

Can you confirm @maryshak1996?